build/kb/Dockerfile

```diff
@@ -14,6 +14,9 @@
 
 FROM python:3.6-slim
 
+RUN apt update \
+  && apt install -y libgl1-mesa-glx libglib2.0-0
```

Comment:
At the end of the dockerfile, clean up the apt cache by `apt clean` and `rm /var/lib/apt/`